.github/workflows/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'bug'
+assignees: ''
+
+---
+
+**Bug description**
+A clear and concise description of what the bug is. Please do not forget to add:
+- Orion version: X.Y.Z
+- MongoDB version X.Y
+- Env variables or CLI parameters
+- Operating System or Docker Image used
+
+**How to reproduce it**
+Steps to reproduce the behavior (as an example)
+1. Send `GET /version`
+2. Create a subscription `POST /v2/subscriptions ...`
```

Comment:
```suggestion
2. Create an entity `POST /v2/entities ...`
```

Both are usual operations, but I think that entity creation is most common that subscription creation, thus better to be use as example sequence in bug.

Reply:
Done in https://github.com/telefonicaid/fiware-orion/pull/4010/commits/6b153302f407775679adbb469c3cfb064acd76e3